src/app/content/components/popUp/FiltersList.tsx

```diff
@@ -65,6 +65,7 @@ export const FiltersListColor = (props: FiltersListColorProps) => (
     <StyledPlainButton
       aria-label={useIntl().formatMessage({id: props.ariaLabelKey(props.color)}, {filterValue: props.color})}
       onClick={props.onRemove}
+      data-analytics-label={`Remove breadcrumb for color ${props.color}`}
     >
```

Comment:
It doesn't work properly for Study guides. Example:
```
aria-label="Remove breadcrumb for label Connections & Relationships"
data-analytics-label="Remove breadcrumb for color green"
```

Reply:
what does the intl message look like? does it have a big condition in there with every label type by color?